finalitygadget/finalitygadget.go

```diff
@@ -256,9 +262,11 @@ func (fg *FinalityGadget) QueryBtcStakingActivatedTimestamp() (uint64, error) {
 	if err != nil {
 		return math.MaxUint64, err
 	}
-
+	fg.logger.Info("All consumer FP public keys", zap.Strings("allFpPks", allFpPks))
 	// check whether the btc staking is actived
 	earliestDelHeight, err := fg.bbnClient.QueryEarliestActiveDelBtcHeight(allFpPks)
+	fg.logger.Info("Earliest active delegation height",
+		zap.Uint64("height", earliestDelHeight))
```

Comment:
we should move the condition at L275-277 to after this line, as `QueryEarliestActiveDelBtcHeight` will throw an error when staking is not yet activated

Reply:
ok